src/ipx.ts

```diff
@@ -20,12 +20,17 @@ export const ipxSetup: ProviderSetup = async (_providerOptions, moduleOptions, n
     .find((mw: { path: string }) => mw.path && mw.path.startsWith('/_ipx'))
 
   if (!hasUserProvidedIPX) {
-    const { createIPX, createIPXMiddleware } = await import('ipx')
-    const ipx = createIPX(ipxOptions)
-    nuxt.options.serverMiddleware.push({
-      path: '/_ipx',
-      handle: createIPXMiddleware(ipx)
-    })
+    try {
+      const { createIPX, createIPXMiddleware } = await import('ipx')
```

Comment:
Can we please catch error here by chaining import promise? This way we don't show a misleading message if an error happens on `createIPX` call.

Reply:
`createIPX` will fail either way, no? I'm not 100% sure what you mean with that.

Reply:
The second line (L25) is also in try/catch. If an error (other than `ipx` is not installed) when creating ipx instance, it will wrongly show a message that ipx is not installed.